.travis.yml

```diff
@@ -5,3 +5,10 @@ rvm:
   - 2.5.7
   - 2.6.5
   - 2.7.0
+  - 2.7.5
+  - 3.0.0
```

Comment:
I guess with one line for each of the middle version would be OK?
Meaning, you can remove:
```
2.7.0
3.0.0
3.1.0
3.2.0
```

Reply:
Also, maybe you can upgrade the bundler version which is way toooo old 😅